libs/cuda_support/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+if (NOT TARGET hpx::cuda)
+  message(FATAL_ERROR "Need cuda")
```

Comment:
Please use `hpx_error()` instead of `message()`, also please make the error message less terse. Perhaps something along the lines of 
> In order to use the CUDA support module, please enable CUDA support while configuring HPX (add `-DHPX_WITH_CUDA=ON` to the cmake command line).

Reply:
That was left in by mistake. I'll improve that